components/inputs/input-text.js

```diff
@@ -307,17 +329,17 @@ class InputText extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))) {
 						title="${ifDefined(this.title)}"
 						type="${this._getType()}"
 						.value="${this.value}">
-					<div id="first-slot"><slot name="${firstSlotName}" @slotchange="${this._onSlotChange}"></slot></div>
```

Comment:
Rename these divs and include the unit within them, which makes measurement for the padding offset easier.